app/services/pro_membership_manager.rb

```diff
@@ -44,8 +44,23 @@ def cancel!
   # Renew the user's pro membership. Make sure the plan they are on is recurring
   # first. If we don't succeed in charging the user then send a dunning email
   # and let their membership expire.
-  def renew!
-    # TODO we don't actually need this until one month after we launch PRO.
+  def renew!(attempt_number: 0)
```

Comment:
Should we cap the `attempt_number`? Since we don't have email templates beyond a certain point?